src/lib/sw_engine/tvgSwRenderer.cpp

```diff
@@ -395,7 +395,7 @@ bool SwRenderer::beginComposite(Compositor* cmp, CompositeMethod method, uint32_
 }
 
 
-Compositor* SwRenderer::target(const RenderRegion& region)
+Compositor* SwRenderer::target(const RenderRegion& region, const RenderRegion* clearRegion)
```

Comment:
@hermet could you please advise a better way of doing this? 
I don't want to change the size of the cmp, since there is no point of rastering anything more than its bbox.
But I need to know at this point also the size of the whole object, that will be masked,  to clear the buffer correspondingly